framework/src/partitioner/LibmeshPartitioner.C

```diff
@@ -82,3 +82,54 @@ LibmeshPartitioner::getPartitioner()
 {
   return _partitioner;
 }
+
+UniquePtr<Partitioner>
+LibmeshPartitioner::clone() const
+{
+//  UniquePtr<Partitioner> cloned_partitioner;
+  switch (_partitioner_name)
+  {
+  case -2: // metis
```

Comment:
Not really me badmouthing METIS. Whoever coded the partitioner MooseEnum in MooseMesh had very little confidence in Metis. What is the deal with starting at -2. though?